test/rails_integration_test.rb

```diff
@@ -26,29 +27,32 @@ def test_simple_get
     def test_x_request_id
       get(
         "/",
-        {},
-        "X-Request-Id" => "external-uu-rid",
-        "X-Request-Depth" => "0",
-        "X-Tree-Path" => "AA"
+        params: {},
+        headers: {
+          "X-Request-Id" => "external-uu-rid",
+          "X-Request-Depth" => "0",
+          "X-Tree-Path" => "AA"
+        }
```

Comment:
Would be nice to add a test for the tracking_id with cases like both headers come on the request the tracking id should have the priority.

Reply:
@bl-rodrigo updated